_site/index.html

```diff
@@ -510,6 +510,20 @@ <h3>T-Z</h3>
           <p class="word__definition">
             the fear, hatred, and stigmitazation of fat people.<br><br>
           </p>
+          <div class="block__type">
+            <p class="word__type">
+              Issues
+            </p>
+            <p class="word__breakdown">It perpetuates the idea that there is a “normal” body type, that there is a direct correlation between thinness and heathliness (which also perpetuates healthiness as an indicator of value), and that fatness is both
+            abnormal and unhealthy.</p>
+          </div>
+          <div class="block__type">
+            <p class="word__type">
+              Impact
+            </p>
+            <p class="word__breakdown">Fat people have a difficult time accessing medical care, they earn less money, are less likely to be believed when reporting sexual assaults, and have
```

Comment:
I understand what you mean here, but I think that more connecting of the dots would be helpful through rephrasing, such as "More hurdles are placed before fat people such as X, which prevent them from accessing medical care, earning less money, etc." 

I want to ensure the systems of oppression that prevent them are explicitly stated.

Reply:
I've updated the `Impact` section to outline the specific facets of fatphobia used by the three different institutions I brought up and how they use them against fat people.